src/tools/auth0/handlers/scimHandler.ts

```diff
@@ -0,0 +1,228 @@
+		return `https://${ this.config('AUTH0_DOMAIN') }/api/v2/connections/${ connection_id }/scim-configuration`;
+	}
+
+	/**
+	 * Creates a new `SCIM` configuration.
+	 */
+	async createScimConfiguration({ id: connection_id }: scimRequestParams, { user_id_attribute, mapping }: scimBodyParams): Promise<AxiosResponse> {
+		log.debug(`Creating SCIM configuration for connection ${ connection_id }`);
+		const url = this.getScimEndpoint(connection_id);
+		return (await this.scimHttpRequest('post', [ url, { user_id_attribute, mapping } ])).data;
+	}
+
+	/**
+	 * Retrieves `SCIM` configuration of an enterprise connection.
+	 */
+	async getScimConfiguration({ id: connection_id }: scimRequestParams): Promise<scimBodyParams> {
+		log.debug(`Getting SCIM configuration from connection ${ connection_id }`);
+		const url = this.getScimEndpoint(connection_id);
+		return (await this.scimHttpRequest('get', [ url ])).data;
+	}
+
+	/**
+	 * Updates an existing `SCIM` configuration.
+	 */
+	async updateScimConfiguration({ id: connection_id }: scimRequestParams, { user_id_attribute, mapping }: scimBodyParams): Promise<AxiosResponse> {
+		log.debug(`Getting SCIM configuration on connection ${ connection_id }`);
+		const url = this.getScimEndpoint(connection_id);
+		return (await this.scimHttpRequest('patch', [ url, { user_id_attribute, mapping } ])).data;
+	}
+
+	/**
+	 * Deletes an existing `SCIM` configuration.
+	 */
+	async deleteScimConfiguration({ id: connection_id }: scimRequestParams): Promise<AxiosResponse>  {
+		log.debug(`Getting SCIM configuration of connection ${ connection_id }`);
+		const url = this.getScimEndpoint(connection_id);
+		return (await this.scimHttpRequest('delete', [ url ])).data;
+	}
+  
+	async updateOverride(requestParams: scimRequestParams, bodyParams: Asset) {
+		// Extract `scim_configuration` from `bodyParams`.
+		// Remove `scim_configuration` from `bodyParams`, because `connections.update` doesn't accept it.
+		const { scim_configuration: scimBodyParams } = bodyParams;
+		delete bodyParams.scim_configuration;
+
+		// First, update `connections`.
+		const updated = await this.connectionsManager.update(requestParams, bodyParams);
+		const idMapEntry = this.idMap.get(requestParams.id);
+
+		// Now, update `scim_configuration` inside the updated connection.
+		// If `scim_configuration` exists in both local and remote -> updateScimConfiguration(...)
+		// If `scim_configuration` exists in remote but local -> deleteScimConfiguration(...)
+		// If `scim_configuration` exists in local but remote -> createScimConfiguration(...)
+		if (idMapEntry?.hasConfig) {
+			if (scimBodyParams) {
+				await this.updateScimConfiguration(requestParams, scimBodyParams);
+			} else {
+				await this.deleteScimConfiguration(requestParams);
+			}
+		} else if (scimBodyParams) {
+			await this.createScimConfiguration(requestParams, scimBodyParams);
+		}
+
+		// Return response from connections.update(...). 
+		return updated;
+	}
+
```

Comment:
We can add a comment, explaining the general intention of this override function

Reply:
This section contains more details about `updateOverride`. https://github.com/auth0/auth0-deploy-cli/pull/921/files#diff-93f0061c303ad4fb91c8a13cd8a3486ca2158f6600bf55a8233d4098ca04c7f7R99